packages/drivekit_core/drivekit_core/example/android/gradle/wrapper/gradle-wrapper.properties

```diff
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.7-all.zip
```

Comment:
nom du commit: "build(ios examples): run ios examples to update native files"

modification du commit: android/build.gradle ... 🫠